addons/gut/signal_watcher.gd

```diff
@@ -151,7 +151,7 @@ func clear():
 	for obj in _watched_signals:
 		if(_utils.is_not_freed(obj)):
 			for signal_name in _watched_signals[obj]:
-				obj.disconnect(signal_name, self, '_on_watched_signal')
+				obj.disconnect(signal_name,Callable(self,'_on_watched_signal'))
```

Comment:
I'm not sure if this will work or not, but it is closer than it was.  This could use a `# !4.0 comment`.

Reply:
Same here, no idea why the tool decided to use Callable here. We can reference the signal directly here